code/modules/mob/living/simple_animal/abnormality/waw/greed_king.dm

```diff
@@ -197,7 +197,7 @@
 
 /* Work effects */
 /mob/living/simple_animal/hostile/abnormality/greed_king/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
-	if(prob(40))
+	if(prob(15))
```

Comment:
Please update the info to reflect that the chance is quite small and not a "normal" probability which is around 50%.